validator/src/cli/thread_args.rs

```diff
@@ -302,6 +336,41 @@ impl ThreadArg for RocksdbFlushThreadsArg {
     }
 }
 
+struct TpuTransactionForwardReceiveThreadArgs;
+impl ThreadArg for TpuTransactionForwardReceiveThreadArgs {
+    const NAME: &'static str = "tpu_transaction_forward_receive_threads";
+    const LONG_NAME: &'static str = "tpu-transaction-forward-receive-threads";
+    const HELP: &'static str =
+        "Number of threads to use for receiving transactions on the TPU fowards port";
+
+    fn default() -> usize {
+        get_max_thread_count()
```

Comment:
let's define a default constant in streamer/quic, DEFAULT_RECEIVER_THREADS = 8 and use that.

Reply:
> let's define a default constant in streamer/quic, DEFAULT_RECEIVER_THREADS = 8 and use that.

Sure, I can move this to streamer. However, I would like this PR to maintain the existing behavior that is creating N-threads threads; use of `get_max_thread_count()` does this.

For example, suppose we encounter a performance regression down the road and call into question whether the change to 8 threads is valid or not. By splitting the value change from the plumbing, we get more granularity should we need to back something out